### DISCUSSION TITLE
CreateButton: apply form field value and redirect parameters

### DISCUSSION THREAD

**Discussion by manosbatsis** [refinedev/refine#5277] [Q&A] [answered]
I'm using a link to [set filters](https://github.com/refinedev/refine/discussions/3007) for my list view as follows:

```
<Link to={`/posts?filters[0][field]=category.id&filters[0][operator]=eq&filters[0][value]=${row.id}`}>
    {filteredPosts?.length}
</Link>
```

This works for my list view by simply adding:

```
const { dataGridProps } = useDataGrid<IPost>({
    syncWithLocation: true,
})
```
How can I use the `CreateButton` in the same list view to:

1. Set the value for a form field using the filter
2. Redirect to the list view after save, using the same filters

**Comment by BatuhanW** [comment, marked as answer]
Hello @manosbatsis, from what I understand, we don't have this feature yet. You can create a feature request. If i get it right, you might accomplish by using `useGo` hook but I'm not quite sure.

**Reply by manosbatsis** [reply]
Thank you @BatuhanW, added https://github.com/refinedev/refine/issues/5286
